pkg/bindings/images/types_diffoptions.go

```diff
@@ -0,0 +1,86 @@
+package images
```

Comment:
Suggestion: Can you stuff an `_` between `diff` and `options` in the filenames?

Reply:
+1